runtime/doc/options.txt

```diff
@@ -6886,8 +6886,6 @@ A jump table for the options with a short description can be found at |Q_op|.
 	   options	options and mappings local to a window or buffer (not
 			global values for local options)
 	   localoptions same as "options"
-	   slash	|deprecated| Always enabled. Uses "/" in filenames.
-	   unix		|deprecated| Always enabled. Uses "\n" line endings.
```

Comment:
I don't see why we need to remove these. They are useful for explicitness.

Reply:
We've removed all instances of one-liner descriptions of deprecated options in `options.txt` before. Unsure why these are special tbh.

Reply:
I still don't get it. If we *remove* a flag of an option they should not be mentioned in `options.txt`, but if we *deprecate* a flag they should be? This seems inconsistent.

I will revert the change in any case.

Reply:
Also hard disagree, deprecation texts like this are obnoxious and unhelpful and I hate it. Dunno if that's a technical reason enough, but 99/10 times my instinctive response to reading these is "Cool, this information is completely useless. Why are you telling me about it? I want my 5 seconds of time back."

Reply:
Alternatively we could just mention the "/" and `\n` in the option description?

Reply:
This is fine for now, let's evaluate it on a future PR 👍